python-code/set-sample/set-sample1.py

```diff
@@ -1,7 +1,8 @@
 import os
 import math
-import sys
```

Comment:
This is a  Unused import sys
```suggestion 
```

 from our GitHub Enhacement Bot